public/graphql/email/send.graphql

```diff
@@ -0,0 +1,9 @@
+mutation email_send($data: HashObject!, $template: String!){
```

Comment:
We don't need to add name to the mutation.
```suggestion
mutation ($data: HashObject!, $template: String!){
```

Reply:
yeh, i know, i just thought it's a nice thing to add (as it doesn't hurt).

Reply:
and in the future we might want to always have a name, there was an idea back at a day to allow to include multiple queries in one file, similarly how you can do it via `pos-cli gui serve` -> graphql editor ;)